compress.c

```diff
@@ -91,10 +91,10 @@ z_size_t Z_EXPORT PREFIX(compressBound)(z_size_t sourceLen) {
         return complen + ZLIB_WRAPLEN;
 
 #ifndef NO_QUICK_STRATEGY
-    /* Quick deflate strategy worse case is 9 bits per literal, rounded to nearest byte,
-       plus the size of block & gzip headers and footers */
-    return sourceLen + ((sourceLen + 13 + 7) >> 3) + 18;
+    /* Quick deflate strategy worst case is 9 bits per literal, rounded to nearest byte,
+       plus the size of block headers, padding and wrappers */
```

Comment:
The comment is a bit confusing.

Reply:
Come to think of it, perhaps the 7 bits was not for the deflate blocks but rather to make sure the bits conversion of sourceLen is rounded up to the next byte. I think that is the case actually, and that should stay as a 7, but that the deflate block header bits also needs to be rounded up to the next byte. I'll amend the commit with that change.

Reply:
And that still passes the test 😄

Reply:
I split up the calculation a little, so it is easier to read, this way it also matches the comment a little better.